framework/src/modules/nft/method.ts

```diff
@@ -49,25 +48,16 @@ import { SetAttributesEvent } from './events/set_attributes';
 
 export class NFTMethod extends BaseMethod {
 	private _config!: ModuleConfig;
-	private _interoperabilityMethod!: InteroperabilityMethod;
 	private _internalMethod!: InternalMethod;
 	private _feeMethod!: FeeMethod;
-	private _tokenMethod!: TokenMethod;
 
 	public init(config: ModuleConfig): void {
 		this._config = config;
 	}
 
-	public addDependencies(
-		interoperabilityMethod: InteroperabilityMethod,
-		internalMethod: InternalMethod,
-		feeMethod: FeeMethod,
-		tokenMethod: TokenMethod,
-	) {
-		this._interoperabilityMethod = interoperabilityMethod;
+	public addDependencies(internalMethod: InternalMethod, feeMethod: FeeMethod) {
 		this._internalMethod = internalMethod;
 		this._feeMethod = feeMethod;
-		this._tokenMethod = tokenMethod;
 	}
 
 	public getChainID(nftID: Buffer): Buffer {
```

Comment:
The logic of `getChainID`, `getLockingModule`, and `getNFTOwner` is now all contained in (to be implemented) `getNFT`. cc @shuse2

Reply:
`getNFT` is now already implemented in [PR](https://github.com/LiskHQ/lisk-sdk/pull/9034). Accordingly `getLockingModule` and `getNFTOwner` are also removed.